### PR TITLE
Add fyle_org_name column to users table and include it in all trackers

### PR DESCRIPTION
**Pull request by jatzz10**
### Issue
For teams app, we don't send `fyle_org_name` as part of any trackers in segment

### Fix
1. Add new column `fyle_org_name` in `users` table
2. Run django migrations and apply them (new columns would have been added)
3. Since, already existing old records will not be having any value in `fyle_org_name` column, we will have to manually fetch the org-name for all existing records and update (add) the org-name for all records.

### CODE REVIEW

File: fyle_teams_app/migrations/0002_user_fyle_org_name.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        ('fyle_teams_app', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='user',
+            name='fyle_org_name',
+            field=models.CharField(blank=True, max_length=120),
+        ),
+    ]
```
Comment (jatzz10) [inline fyle_teams_app/migrations/0002_user_fyle_org_name.py:18]: this file is auto-generated by django after when we -
1. go into the teams-app service in local (`docker-compose exec teams-app-service bash`)
2. run migrations there (`python manage.py makemigrations`)

**Comment by gokulk16** [comment]
> 3\. Since, already existing old records will not be having any value in `fyle_org_name` column, we will have to manually fetch the org-name for all existing records and update (add) the org-name for all records.

can you add the specific script to run for this also in the same PR.
can create a new folder called specific_DB_migrations and add the SQL script into it

**Review by gokulk16 — CHANGES_REQUESTED** [review]
neededd the sql script to populate old records' org_name.

File: scripts/add_fyle_org_name_migration_changes.sql [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+------------------------------------------------------------------------------
+
+
+-- Script to take dump of all distinct fyle-org-ids from slack db
+\copy (
+    select 
+        distinct fyle_org_id
+    from
+        users
+) to 'slack_orgs.csv' csv header;
```
Comment (gokulk16) [inline scripts/add_fyle_org_name_migration_changes.sql:14]: these are teams orgs no?
if yes, then can rename file to `teams_orgs.csv`

File: scripts/add_fyle_org_name_migration_changes.sql [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        id in (
+            select
+                id
+            from
+                slack_orgs
+        )
+) to 'slack_org_names.csv' csv header;
+
+
+-- Script to update the 'fyle_org_name' column with respective org names
```
Comment (jatzz10) [inline scripts/add_fyle_org_name_migration_changes.sql:38]: mention teams db

Comment (jatzz10) [reply scripts/add_fyle_org_name_migration_changes.sql:38]: add specific scripts for updating

File: scripts/add_fyle_org_name_migration_changes/fetch_org_names_from_prod_dbs.sql [hunk 4]
```diff
@@ -0,0 +1,9 @@
+-- Script to take the dump of org-names for all orgs, inside both prod dbs
+\copy (
+    select
```
Comment (gokulk16) [inline scripts/add_fyle_org_name_migration_changes/fetch_org_names_from_prod_dbs.sql:3]: selecting all the orgs is a huge operation. can we do some filtering here itself?

**Review by gokulk16 — CHANGES_REQUESTED** [review]

File: scripts/add_fyle_org_name_migration_changes/fetch_org_names_from_prod_dbs.sql (hunk 4 above)
Comment (gokulk16) [reply scripts/add_fyle_org_name_migration_changes/fetch_org_names_from_prod_dbs.sql:3]: we can dump the teams_db_orgs into a table in here and pull only the needed data. can u try that

**Review by gokulk16 — APPROVED** [review]

**Pull request merged**